Makefile.am

```diff
@@ -1,40 +1,56 @@
 
-bin_SCRIPTS =
+bin_PROGRAMS =
+BUILT_SOURCES =
 CLEANFILES =
 DISTCLEANFILES =
 EXTRA_DIST =
 
 resource_files = $(shell $(GLIB_COMPILE_RESOURCES) --sourcedir=$(srcdir) --generate-dependencies $(srcdir)/speedwagon.gresource.xml)
-speedwagon.gresource: speedwagon.gresource.xml $(resource_files)
-	$(AM_V_GEN)$(GLIB_COMPILE_RESOURCES) --target=$@ --sourcedir=$(srcdir)  $<
-CLEANFILES += speedwagon.gresource
-
-do_subst = sed \
-	-e 's|@bindir[@]|$(bindir)|g' \
-	-e 's|@pkglibdir[@]|$(pkglibdir)|g' \
-	-e 's|@pkgdatadir[@]|$(pkgdatadir)|g' \
-	-e 's|@GJS_CONSOLE[@]|$(GJS_CONSOLE)|g'
-
-src/config.js: src/config.js.in Makefile
-	$(AM_V_GEN) $(MKDIR_P) $(builddir)/src && \
-	$(do_subst) $< > $@
-CLEANFILES += src/config.js
-
-js_sources = \
-	src/config.js \
-	src/eos_speedwagon.js \
+src/speedwagon-resources.h: speedwagon.gresource.xml Makefile $(resource_files)
+	$(AM_V_GEN)$(GLIB_COMPILE_RESOURCES) --target=$@ --sourcedir=$(srcdir) --generate-header --c-name eos_speedwagon $<
+src/speedwagon-resources.c: speedwagon.gresource.xml Makefile $(resource_files)
+	$(AM_V_GEN)$(GLIB_COMPILE_RESOURCES) --target=$@ --sourcedir=$(srcdir) --generate-source --c-name eos_speedwagon $<
```

Comment:
Not critical, but this will make the resources be compiled twice. You can just express that one target is a side effect of the other. E.g., keep the `src/speedwagon-resources.h` rule, but then just say that `src/speedwagon-resources.c` depends on it without adding a recipe:
```
src/speedwagon-resources.c: src/speedwagon-resources.h
```